src/App/App.js

```diff
@@ -15,69 +14,71 @@ class App extends Component {
      movies: [],
      currentSelectionId: null,
      error: null,
-     dummyDisplay: {
-      id: 1, 
-      title: "Fake Movie Title", 
-      poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg", 
-      backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg", 
-      release_date: "2019-12-04", 
-      overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!", 
-      average_rating: 6, 
-      genres: ["Drama"], 
-      budget:63000000, 
-      revenue:100853753, 
-      runtime:139, 
-      tagline: "It's a movie!" 
-      }   
+     selectedMovie: null
    }
  }
 
  componentDidMount = () => {
-  fetchData('moves').then(data => {
+  fetchData('movies').then(data => {
     this.setState({movies: [...this.state.movies, ...data.movies]})
-  }).catch((data) => {console.log("I made it to catch", data)
-})
+  }).catch((data) => this.setState({error: data.message}))
```

Comment:
This would be helpful to revisit so down the line, especially if we include other types of fetch requests. We might want to write our own error messages for the user based on the error.